### ISSUE TITLE
Generated HTML does not contain <!DOCTYPE html>

### ISSUE THREAD

**Issue by patrickfav** [CONNECT-platform/codedoc#10]
Found while checking my site with Chromes Lighthouse feature:

https://web.dev/doctype/?utm_source=lighthouse&utm_medium=devtools

The currently generated HTML lack `<!DOCTYPE html>`

**Comment by loreanvictor** [comment]
fixed by [this commit](https://github.com/CONNECT-platform/codedoc/commit/9f4b2363230cc60d0d5e759910f3777c194c62a0), deployed via `@codedoc/core@0.1.45`. @patrickfav thanks for the heads up!

**Comment by patrickfav** [comment]
Cool, thanks for the quick response! Awesome project btw, one of the best/modern static site generators Ive seen (Jekyll is of course waaaaayyy more feature rich, but thats to be expected). I kinda "misue" it a bit, because I use it as a personal website with all the readmes of my github pages & medium articles - but so for it works fine :)

**Comment by loreanvictor** [comment]
@patrickfav thanks! and to be honest thats not a misuse. I mean I started this because I had a lot of open-source documentation to write, but then people started using it as a blog and so we (with the rest of the team) even started [an open-source blog platform](https://coding.blog) based on it.

And yep this project is about 2.5 weeks old (about a month and a half since I started working on it), but anytime you miss a Jekyll's feature, don't hesitate to open an issue (or ideally a PR ;) ).

**Comment by patrickfav** [comment]
2.5 weeks? Thats exceptional, really! Very good work. That explains why, starting my site on Friday, found a feature in your offical docs on Monday - was not working - reason was, I had to update the cli because you just released it :)

Anyways, if thats ok with you I will add some issues (basically early adopter things), hopefully I will be able to contribute PRs as well ;)

I saw the coding.blog link on pointer.io, thats why I found this project :)
